src/phoebus_driver.cpp

```diff
@@ -1071,9 +1073,7 @@ void UserWorkBeforeOutput(MeshBlock *pmb, ParameterInput *pin) {
         Real gam1[3][3];
         Real gam2[3][3];
         Real gam3[3][3];
-        auto analysis = pmb->packages.Get("analysis").get();
         const Real z = coords.Xc<3>(k, j, i);
-        const Real sigma = analysis->Param<Real>("sigma");
```

Comment:
why are you deleting these lines? I need sigma

Reply:
I moved them out of the `par_for` as this code crashes on device. Packages and params are only allocated on the CPU.

Reply:
oh shoot, I did too in my PR.

Reply:
But I don't think that's a problem, maybe one of us can undo this change and whichever PR is merged first merge that main into another PR. @Yurlungur what do you think?

Reply:
There's no need to undo changes. Whoever's PR gets merged first, git will guide you through a manual merge process if there are merge conflicts on the second PR.

Reply:
Okay then even better